src/ActionKit/Action.php

```diff
@@ -113,6 +114,13 @@ public function __construct(array $args = array(), array $options = array())
 
         // use the schema definitions to filter arguments
         $this->args = $this->_filterArguments($args);
+        
+        session_start();
```

Comment:
this line needs to be in `login.php`.  we should remove the session_start() dependency